src/Unleash/UnleashSettings.cs

```diff
@@ -103,6 +103,7 @@ public class UnleashSettings
         ///
         /// Default: A serializer based on Newtonsoft will be used, given that these assemblies are loaded into the appdomain already.
         /// </summary>
+        [Obsolete("No longer supported in recent versions of Unleash, will be removed in the next major version", false)]
```

Comment:
It's not really "recent versions of Unleash" though, is it? How about something like this instead?

```suggestion
        [Obsolete("This setting is no longer supported in recent versions of the Unleash .NET SDK and will be removed in the next major release.", false)]
```

Reply:
It's just the same deprecation notice that got splashed on a bunch of other things in preparation for the next version. I'm going to keep it consistent